scripts/zones/Lower_Jeuno/Zone.lua

```diff
@@ -81,16 +84,42 @@ end;
 
 function onGameHour()
     local VanadielHour = VanadielHour();
-
+    local playerOnQuestId = GetServerVariable("[JEUNO]CommService");
```

Comment:
I think it does need to persist across server crash.  Otherwise you could get two players flagged for the quest on the same game-day.